packages/conf-assimp/conf-assimp.1/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "geoffderosenroll@gmail.com"
+authors: ["Geoff deRosenroll"]
+homepage: "https://github.com/assimp/assimp"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "BSD-3-Clause"
+build: [
+  ["assimp" "--help"] {os != "openbsd"}
+  ["/usr/local/bin/assimp" "--help"] {os = "openbsd"}
```

Comment:
why is that different on openbsd?

Reply:
I have to admit that I copied that quirk from [conf-pkg-config](https://github.com/ocaml/opam-repository/blob/master/packages/conf-pkg-config/conf-pkg-config.2/opam), which I used as a reference when writing this / learning about *conf* packages.